### DISCUSSION TITLE
Merging data arrays WindowError: bounds and transforms are inconsistent

### DISCUSSION THREAD

**Discussion by nilsleh** [corteva/rioxarray#699] [Q&A]
I am trying to merge some `DataArrays` based on spatial coordinates. The data I am working with is climate data on a global grid. I am running into `rasterio.errors.WindowError: Bounds and transform are inconsistent` when trying to use the `merge_arrays()` function. I recreated the issue in this [google colab notebook](https://colab.research.google.com/drive/1scHihdllLzOavUOHbXurkzfBD8Nh-A60?usp=sharing). The real data I have does not inherently have a transform defined that could be accessed by `rioxarray` so I think there is one created on the fly and I get an error. I am wondering how I should handle my data to make the `merge_array()` work.

**Comment by RichardScottOZ** [comment]
Did write_crs definitely work?

**Reply by nilsleh** [reply]
No, I think we are still having that issue, although that PR for which we need it has gone stale for now https://github.com/microsoft/torchgeo/pull/1490 but will be picked up again.

**Comment by snowman2** [comment]
I believe that this is the source of your issue:

```python
lats = np.linspace(lat_min, lat_max, SIZE)
```
Your raster is upside down. If you reverse your y coordinates in the raster, the bounds will be as expected. Otherwise, you will need to change the input bounds to have miny and maxy swapped.
